build-package.sh

```diff
@@ -500,6 +504,7 @@ _show_usage() {
 	echo "  -o Specify directory where to put built packages. Default: output/."
 	echo "  --format Specify package output format (debian, pacman)."
 	echo "  --library Specify library of package (bionic, glibc)."
+	echo "  --disable-dependency-download-parallelizing disable dependency downloading parallelizing"
```

Comment:
That option name is wayyyyy too long.
How about:
```suggestion
	echo "  --no-parallel disable parallel dependency downloads"
```

Reply:
This case does not specify where exactly parallelizing is to be disabled.